design/one-pager-resource-connectivity-mvp.md

```diff
@@ -638,7 +638,279 @@ parameters:
 
 ### Amazon Web Services
 
-TBD.
+In Amazon Web Services (AWS), `KubernetesCluster` and `MySQLInstance` claims are satisfied by `EKSCluster` and `RDSInstace` resource classes. 
+Similar to GCP, one network and one or more sub-networks are needed to connect `EKSCluster` and `RDSInstance` instances:
+
+* `AwsVpc`: creates a virtual private cloud (VPC) in AWS.
```

Comment:
Can we omit the `Aws` prefix given these will be within an API group that contains `aws`? i.e. `vpc.network.aws.crossplane.io`, `subnet.network.aws.crossplane.io`?

Also, we're not super consistent in the current codebase, but I'd prefer to follow Go style capitalization rules for acronyms; i.e. `VPC` not `Vpc`.